app/lib/github_badges.rb

```diff
@@ -107,19 +107,19 @@ def self.grant_contributor_badges!
     def self.contributor_badges
       unless bronze = Badge.find_by(name: BADGE_NAME_BRONZE)
```

Comment:
Here's how to write the query to be case insensitive 😉

```suggestion
      unless bronze = Badge.where("name ILIKE ?", BADGE_NAME_BRONZE).first
```